### PR TITLE
[MNOE-407] Add config and locales for onboarding wizard

### PR DESCRIPTION
**Pull request by hedudelgado**
@ouranos

### CODE REVIEW

**Comment by ouranos** [comment]
We'll need to review how to store the state so, for now, could you amend this PR to only have:
* the feature flag
* the locales (see [this file](https://github.com/maestrano/mno-enterprise-demo/blob/ca4b13cb3271d66bb8fdb8437d3ab77cf6d3f060/config/locales/en.yml))

**Comment by hedudelgado** [comment]
Hey @ouranos, 
I do not get the locales bit. They are all in mno-enterprise-angular.
https://github.com/maestrano/mno-enterprise-angular/pull/183/files#diff-a59849a302c5e8445a7eeb6dc0ace994R354

File: api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder [hunk 1]
```diff
@@ -12,6 +12,7 @@ json.multi_instantiable app.multi_instantiable
 json.subcategories app.subcategories
 json.average_rating app.average_rating
 json.running_instances_count app.running_instances_count
+json.add_on app.add_on?
```
Comment (ouranos) [inline api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder]: This is leaking from https://github.com/maestrano/mno-enterprise/pull/235

File: api/spec/controllers/mno_enterprise/jpi/v1/marketplace_controller_spec.rb [hunk 2]
```diff
@@ -38,7 +38,8 @@ def partial_hash_for_app(app)
         'multi_instantiable' => app.multi_instantiable,
         'subcategories' => app.subcategories,
         'average_rating' => app.average_rating,
-        'running_instances_count' => app.running_instances_count
+        'running_instances_count' => app.running_instances_count,
+        'add_on' => app.add_on?
```
Comment (ouranos) [inline api/spec/controllers/mno_enterprise/jpi/v1/marketplace_controller_spec.rb]: This is leaking from https://github.com/maestrano/mno-enterprise/pull/235

File: core/config/locales/templates/onboarding/en.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          invoices: "Invoices"
+          journals: "Journals"
+          employees: "Employess"
+          opportunities: "Opportunities"
+          what_data: "What data will be synchronised"
+          start: "Start visualise the data synchronised by your applications by selecting one by clicking on its icon."
+          learn_more: "Learn more on how Maestrano synchronise data between your applications."
+        end_wizard:
+          almost_there: "We’re Almost There"
+          almost_there:
```
Comment (ouranos) [inline core/config/locales/templates/onboarding/en.yml]: You have twice the same keys here. It won't work with YAML

**Comment by hedudelgado** [comment]
@ouranos sorry about the leaks. I did reset, clean and push a commit to clean history.

**Pull request merged**